src/js/base/ripe.js

```diff
@@ -1887,7 +1887,7 @@ ripe.Ripe.prototype._loadInitialsBuilder = async function() {
     });
     // eslint-disable-next-line no-eval
     const logicScript = eval(logicScriptText);
-    this.initialsBuilder = logicScript ? logicScript.initialsBuilder : this._initialsBuilder;
+    this.initialsBuilder = logicScript ? logicScript.initialsBuilder : this._initialsBuilder.bind(this);

```

Comment:
Shouldn't `logicScript.initialsBuilder` be `bind(this)` as well?

Reply:
Yes, but then I would need to change the ripe-compose PR